src/containers/App/App.js

```diff
@@ -34,10 +33,12 @@ class App extends Component {
               }} />
               <Route path='/notes/:id' render={({ match }) => {
                 const { id } = match.params
-                const note = this.props.notes.find(note => note.id === id)
+                const note = this.props.notes.find(note => note.id == id)
```

Comment:
why did you have to change this? I thought it was working with the strictly equals?

Reply:
for some reason the test file is seeing the id as a string when using the mock data